packages/data-explorer-ui/src/components/common/Banner/banner.styles.ts

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled";
+
+export const Banner = styled.div`
+  background-color: ${({ theme }) => theme.palette.primary.main};
+  color: ${({ theme }) => theme.palette.common.white};
+  padding: 8px;
```

Comment:
Insert between lines 5 and 6: `display: flex`

Reply:
And then, after the `display: flex` add `justify-content: center`.

Reply:
There is a line-height on the `body` that the Banner is inheriting; the `display: flex` and alignment centre the content, and resolve the line-height problem (as well as using the correct Typography style for the content).